smallvil/src/input.rs

```diff
@@ -101,12 +97,8 @@ impl Smallvil {
                 );
             }
             InputEvent::PointerAxis { event, .. } => {
-                // TODO: From trait
-                let source = match event.source() {
-                    AxisSource::Continuous => wl_pointer::AxisSource::Continuous,
-                    AxisSource::Finger => wl_pointer::AxisSource::Finger,
-                    AxisSource::Wheel | AxisSource::WheelTilt => wl_pointer::AxisSource::Wheel,
-                };
+                let source: wl_pointer::AxisSource = event.source().into();
```

Comment:
From might be more clear here and in the other instances?

Reply:
Indeed, done.